emp-daisy/HelloBooksLib
# #166358637 Minor bug fixes

#### What does this PR do?
This PR fixes project-wide bugs, install some dependencies and converts the database related files to ES6

#### Description of Task to be completed?
- fix bugs on different files
- modify database files to use ES6
- install morgan to log requests to the server in development
- install debug to replace console.log
- some minor bug fixes
- update `heroku-postbuild` to copy docs file to the build folder after build

#### How should this be manually tested?
- clone the repo and look through the files
- run the server and make a request to see the request logs on the console
- visit the [staging app link](https://helobooks-staging.herokuapp.com/api/v1) @https://helobooks-staging.herokuapp.com/api/v1 and test with postman

#### Any background context you want to provide?
- None
#### What are the relevant pivotal tracker stories?
[#166358637](https://www.pivotaltracker.com/story/show/166358637)

#### Screenshots (if appropriate)
